hosting.html

```diff
@@ -0,0 +1,188 @@
+
+				<h4>DATA CENTERS / COLOCATION</h4>
+				<p>The main alternative to home hosting is to find a data center (also sometimes
+					called a colocation	facility or "colo") where your Obelisks can be hosted.  It can be
+					quite time consuming to find a good facility that understands the needs of miners,
+					provides "remote hands" services if your miner needs attention, and offers a complete
+					package at a good price.</p>
+
+				<h4>OBELISK APPROVED HOSTING</h4>
+				<p>We want owning and operating your Obelisks to be an easy, fun and rewarding experience,
+					so we've done the negotiating for you!  Below you will find a list of approved
+					hosting packages at companies with locations across the United States and Europe, and we're
+					working on adding more international locations.</p>
+				<p>One major benefit of selecting an approved hosting package is that your Obelisks
+					will be shipped directly to your chosen provider. They will receive your Obelisks,
+					unpack them, connect them to power &amp; internet, then send you an email with information
+					on how to login remotely to your Obelisks so you can configure them and start mining.</p>
+
+					<h4>AVAILABLE PACKAGES</h4>
+					<p>The following hosting packages have been negotiated for you by Obelisk.  Soon, you will
+						be able to login to the Obelisk site and choose a provider.</p>
+						<table class="colo-table" width=100%>
+							<thead>
+								<tr class="colo-table-header">
+									<th>PROVIDER NAME</th>
+									<th>LOCATION</th>
+									<th>MONTHLY PRICE</th>
+									<th>PACKAGE DETAILS</th>
+								</tr>
+							</thead>
+							<tbody>
+								<tr>
+									<td>Syptec Krypto</td>
+									<td>Salt Lake City, Utah</td>
+									<td>$60/Obelisk, or<br>$450/8 Obelisks</td>
+									<td>
+										<ul>
+											<li>All power costs included</li>
+											<li>Remote hands included for reboots</li>
+											<li>Emergency Backup Power (3 days)</li>
+											<li>Secure data center specifically for mining</li>
+											<li>Additional services<ul>
+												<li>Private locked rack: $35/month</li>
+												<li>Miner cleaning/dusting: $20/request</li>
+											</ul>
+										</li>
+									</ul>
+								</td>
+							</tr>
+							<tr>
+								<td>Minery.tech</td>
+								<td>Smithville, Missouri</td>
+								<td>$50</td>
+								<td>
+								<ul>
+									<li>All power costs included</li>
+									<li>Remote hands included for reboots</li>
+									<li>No Emergency Backup Power</li>
+									<li>Smaller, but secure data center specifically for mining</li>
+									<li>Additional services<ul>
+										<li>Private locked rack: $35/month</li>
+										<li>Miner cleaning/dusting: $20/request</li>
+									</ul>
+								</li>
+							</ul>
+						</td>
+					</tr>
+				</tbody>		
+			</table>
+			<p>If you	know of a good hosting provider and you would like us to investigate adding an
```

Comment:
The spacing here is a bit odd

Reply:
Looks like a tab snuck in there instead of a space.